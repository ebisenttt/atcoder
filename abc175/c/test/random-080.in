958175831145858 517511545824895 233745504909579
