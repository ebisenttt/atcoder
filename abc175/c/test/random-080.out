210551693402037
